dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/transform/ContentletTransformerJson.java

```diff
@@ -0,0 +1,42 @@
+package com.dotmarketing.portlets.contentlet.transform;
+
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.portlets.contentlet.model.Contentlet;
+import com.dotmarketing.util.Logger;
+import com.dotmarketing.util.UtilMethods;
+import com.liferay.portal.model.User;
+import java.util.HashMap;
+import java.util.Map;
+
+public class ContentletTransformerJson {
+
+    private final Contentlet contentlet;
+
+    public ContentletTransformerJson(final Contentlet contentlet) {
+        this.contentlet = contentlet;
+    }
+
+    public Map<String,Object> toMap(){
+       return contentletToMap(contentlet);
+    }
+
+    private Map<String, Object> contentletToMap(final Contentlet contentlet) {
+        User modUser = null;
+        try {
+            modUser = APILocator.getUserAPI().loadUserById(contentlet.getModUser());
+        } catch (Exception e) {
+            Logger.error(getClass(),"Error loading user from db. ", e);
+        }
+        final Map<String, Object> conMap = new HashMap<>();
+        conMap.put("title", contentlet.getTitle());
+        conMap.put("modDate", UtilMethods.dateToJDBC(contentlet.getModDate()));
```

Comment:
replace by Contentlet.MOD_DATE_KEY, the same for the rest, languageId could be languageId instead of language and you can probably add title to the Contentlet constants